changelog.json

```diff
@@ -1,5 +1,28 @@
 {
   "releases": {
+    "1.3.0": [
+      "[New] Notification displayed in History tab when the base branch moves ahead of the current branch - #4768",
+      "[New] Repository list displays uncommitted changes count and ahead/behind information - #2259",
+      "[Added] Option to move repository to trash when removing from app - #2108. Thanks @say25!",
+      "[Added] Syntax highlighting for PowerShell files - #5081. Thanks @say25!",
+      "[Fixed] \"Discard Changes\" context menu discards correct file when entry is not part of selected group - #4788",
+      "[Fixed] Display local path of selected repository as tooltip - #4922. Thanks @yongdamsh!",
+      "[Fixed] Display something when repository is located at drive root - #4924",
+      "[Fixed] Ensure commit message is cleared after successful commit - #4046",
+      "[Fixed] Handle legacy macOS right click gesture - #4942",
+      "[Fixed] History omits latest commit from list - #5243",
+      "[Fixed] Markdown header elements hard to read in dark mode - #5133. Thanks @agisilaos!",
```

Comment:
Is dark mode available to the production version of the app? If so, that's a 👍. If the dark mode is available only in beta we should remove that.

Reply:
Yes, it's been there since 1.2.4 - we've only marked the Dark Theme as beta in-app because we wanted to make it clear that it's something we're still iterating on, and to provide feedback.

Reply:
👍🏻. I’m using the beta so I wasn’t sure. 😂